### ISSUE TITLE
[WorkwebUI] Add AUTH support for password protected redis servers.

### ISSUE THREAD

**Issue by darkpssngr** [gocraft/work#114]
Right now its not possible to start the WebUI when the redis servers need auth. Can we add an option call --redisPassword and update the dialer to use it if its present.

**Comment by chemdrew** [comment]
Looks like the connection string is being parsed using IANA specification (https://www.iana.org/assignments/uri-schemes/prov/redis)

I didn't use a username in auth so for me it was just this command to get it running
```
workwebui -redis="redis://:my_super_secret_password@redis.example.com:6379"
```

**Comment by rivernews** [comment]
> Looks like the connection string is being parsed using IANA specification (https://www.iana.org/assignments/uri-schemes/prov/redis)
> 
> I didn't use a username in auth so for me it was just this command to get it running
> 
> ```
> workwebui -redis="redis://:my_super_secret_password@redis.example.com:6379"
> ```


Yeah, for anyone who got the error `ERR wrong number of arguments for 'AUTH' command`, make sure you remove the username portion of it as @chemdrew did.  Not sure why username causes trouble, I'm using Redis Cloud addon from Heroku which provides a username in the redis URL. But as long as I remove username I don't get any trouble accessing redis now.
